### ISSUE TITLE
Rules: Update Rules

### ISSUE THREAD

**Issue by mikekasprzak** [JammerCore/JammerCore#609]
As a spin off of #575, here is a new issue for makings requests for changes and added to the rules.

https://ldjam.com/events/ludum-dare/rules/

The original rules page is here:

http://ludumdare.com/compo/rules/

The initial draft of the rules ported to markdown from the old site are by @jezzamonn :+1: 

https://gist.github.com/Jezzamonn/4be04550318eefaed85e5e97c3ac126b

Fixing #608 would benefit future changes to the page.

---

Also #469 is similar to this: a guide on how to host your games.

https://ldjam.com/events/ludum-dare/hosting-guide/

**Comment by povrazor** [comment]
I've made the following change on the live site. The link to the playlist of samples is this:

https://soundcloud.com/ludumdare/sets/derivative-work-example

I also looked in to getting the Soundcloud embeds working, and it turns out this is a much bigger hassle than I initially thought. 

Unlike Twitch and YouTube, Soundcloud embeds are quite exotic. They use very nonstandard urls, and can't be derived by chopping up the URL.

On the plus, Soundcloud does have an oembed endpoint that you can feed an original URL to. Several sites seem to support this (Twitter included), but as soon as we let 3rd party sites inject code, it needs to be sandboxed. #545

YouTube embedding is the only exception, but it's temporary, because we really need YouTube support. Other embeds we can live without.

**Comment by povrazor** [comment]
An alternative to the Soundcloud embeds would be if we abandoned Soundcloud entirely for audio, and hosted mp3/ogg files ourselves, and wrapped them in our own HTML5 audio player.

Frankly I have not been too happy with Soundcloud. An experiment I tried a while ago was building a sort of "Ludum Dare Radio" client using Soundcloud as the backend.

http://jammer.tv/radio.php

It worked for the most part, but frustratingly 1 of my 3 test tracks would not play in Chrome (Firefox was fine). Now it appears none of the tracks will play in Chrome.

...

Yeah, this is probably something we should do. I've already been looking in to making our own version of GIFV (i.e. silent MP4's). We should be capable of hosting all our own media, just with smaller max file sizes than 3rd parties.

Plus though only a few people will ever use it, I kinda want **LD FM**. ;)
